assembly/src/assembler/instruction/field_ops.rs

```diff
@@ -7,6 +7,14 @@ use crate::MAX_EXP_BITS;
 /// Field element representing TWO in the base field of the VM.
 const TWO: Felt = Felt::new(2);
 
+// ASSERTIONS
+// ================================================================================================
+
+/// Asserts that the top two words in the stack are equal.
```

Comment:
MIght be worth adding the cycle count to this comment as well.

Reply:
added